buildpack.toml

```diff
@@ -11,10 +11,25 @@ api = "0.7"
     uri = "https://github.com/paketo-buildpacks/node-engine/blob/main/LICENSE"
 
 [metadata]
-  include-files = ["bin/build", "bin/detect", "bin/run", "bin/optimize-memory", "bin/inspector", "buildpack.toml"]
-  pre-package = "./scripts/build.sh"
-  [metadata.default-versions]
-    node = "20.*.*"
+  include-files = [
+      "buildpack.toml",
+      "linux/amd64/bin/build",
+      "linux/amd64/bin/detect",
+      "linux/amd64/bin/run",
+      "linux/amd64/bin/optimize-memory,
+      "linux/amd64/bin/inspector,
```

Comment:
Seems like there is a double quote missing here. 
Are there any errors that pack throws after fixing the double quote?